packages/odyssey-storybook/src/components/TagList/TagList.mdx

```diff
@@ -1,28 +1,28 @@
 import { Canvas, Story, ArgsTable } from "@storybook/addon-docs";
```

Comment:
I tweaked this a little, might be worth opening the entire file and verifying that it all still make sense.